### DISCUSSION TITLE
Malfunction after update to v11.3.1

### DISCUSSION THREAD

**Discussion by Drillo123** [jomjol/AI-on-the-edge-device#1081] [General]
Since the update, only nonsensical values are calculated.
RAW Value recognizes e.g. 28072.81 as a value then 421051.060 is returned.

What can it be?

Here is an excerpt from the .ini:

![grafik](https://user-images.githubusercontent.com/78371030/192216407-9524bef3-3aa3-42ea-ae22-bef8bcc23936.png)

**Comment by friedpa** [comment]
Please send extended config.ini (Expert Mode). But without the Paswords :o)

**Reply by caco3** [reply]
password is not part of that ini file!

**Comment by friedpa** [comment]
MQTT Password.....

**Reply by caco3** [reply]
> MQTT Password.....

right, thanks! => I created https://github.com/jomjol/AI-on-the-edge-device/issues/1092 to improve this

**Comment by jomjol** [comment]
The newest Rolling has solved the problem with the password in log files and serial output. 
But the effort to remove the mqtt password from the config.ini is too high in my view.
